example/android/app/src/main/java/com/pspdfkit/flutter/example/FlutterExampleApplication.java

```diff
@@ -0,0 +1,22 @@
+/*
+ *   Copyright © 2018-2021 PSPDFKit GmbH. All rights reserved.
+ *
+ *   THIS SOURCE CODE AND ANY ACCOMPANYING DOCUMENTATION ARE PROTECTED BY INTERNATIONAL COPYRIGHT LAW
+ *   AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE AGREEMENT.
+ *   UNAUTHORIZED REPRODUCTION OR DISTRIBUTION IS SUBJECT TO CIVIL AND CRIMINAL PENALTIES.
+ *   This notice may not be removed from this file.
+ */
+package com.pspdfkit.flutter.example;
+
+import android.content.Context;
+
+import androidx.multidex.MultiDex;
+import io.flutter.app.FlutterApplication;
+
+public class FlutterExampleApplication extends FlutterApplication {
+    @Override
+    protected void attachBaseContext(Context base) {
+        super.attachBaseContext(base);
+        MultiDex.install(this);
+    }
+}
```

Comment:
@amit-nayar I would not add a separate class file for this. Since Flutter no longer expects applications to use the `FlutterApplication` base class, we can directly refer to the `MultiDexApplication` class within the manifest, which allows us to remove this file entirely. The `MultiDexApplication` will then call `MultiDex.install()` for us. Does this make sense? Please let me know if you have other questions about this.